api/routes/web.php

```diff
@@ -15,3 +15,15 @@
 
 Route::get('/', 'HomeController@index')->name('home');
 Route::get('/home', 'HomeController@index')->name('home');
+
+Route::get('/user/{id}/profile', 'UserController@show')->name('user.profile');
```

Comment:
It was possible to use resource method with except/only , but it is also descent way

Reply:
Since it's not a lot of routes, I'd like to have the specific links in one file, but yes, resource could've been used here.